public-docs/swag-shop-03-custom-styling.md

```diff
@@ -0,0 +1,61 @@
+---
+id: swag-shop-3
+title: Custom Styling
+---
+
+# Adding custom styling
+
+## Styling a existing component
+All the CSS styles can be added directly in `lib/reactionTheme.js` file.
+For example:
+To style the Header component the steps are:
+1. Open the Header component at `/src/components/Header/Header.js` and take note of the `name` of the Component passes to `withStyle`. Here it is "Header" as is evident from the line `@withStyles(styles, { name: "Header" })`
```

Comment:
- Link to the file at v0.1.0 - [`/src/components/Header/Header.js`](https://github.com/reactioncommerce/reaction-next-starterkit/blob/v0.1.0/src/components/Header/Header.js) 

- Looks like this [line](https://github.com/reactioncommerce/reaction-next-starterkit/blob/v0.1.0/src/components/Header/Header.js#L40) has changed already here, to `@withStyles(styles, { name: "SkHeader" })`

I'm guessing `Sk` starts for Starterkit(??)

Reply:
@machikoyasuda  @zenweasel  `Sk` is for starter kit.

Reply:
None of this will be linked directly to the sk repo. There will be a separate repo for this tutorial as mentioned in the description

Reply:
Ooo!!! Nevermind~